panoptes_client/panoptes.py

```diff
@@ -351,8 +351,7 @@ def get_csrf_token(self):
         return self.session.get(url, headers=headers).headers['x-csrf-token']
 
     def get_bearer_token(self):
-        check_time = datetime.now() - timedelta(minutes=1)
-        if not self.bearer_token or self.bearer_expires < check_time:
+        if not self.bearer_token or self.bearer_expired():
```

Comment:
wondering if this could be combined into `if self.valid_bearer_token?(self)` to simply the conditional?